src/main/java/frc/robot/commands/drive/Drive.java

```diff
@@ -15,24 +20,60 @@
      */
     private boolean sideFlipped = false;
 
+    private double pitchDegrees;
+
     public Drive() {
         requires(Drivetrain.getInstance());
     }
 
+    private void correctForPitch(double[] stickSpeeds) {//x-tip
```

Comment:
Would it be better form to return the new list instead of modifying the one passed? I may be stuck in C++ thinking... :thinking:

Reply:
It works. :man_shrugging: